### ISSUE TITLE
Relay Entrypoints Documentation

### ISSUE THREAD

**Issue by outerlook** [facebook/relay#3150]
Hi!

Didn't find it as an issue, so I think it would be nice to have one for tracking "Relay Entrypoints API" documentation situation.

https://relay.dev/docs/en/a-guided-tour-of-relay#preloading-data-for-transitions-in-parallel-with-code-client-preloading
> In order to do so, we can use Relay EntryPoints, which are a set of APIs for efficiently loading both the code and data dependencies of any view in parallel. Check out our api reference for Entry Points:

It raises some interest on solving this particular parallelization problem. Maybe I didn't get it right, but I didn't find its documentation anywhere. Is this too experimental, and will be improved before releasing some examples?

**Comment by josephsavona** [comment]
Yup, this feature is currently not covered by the Relay docs; we plan to include EntryPoints when the `relay-experimental` package is stable. For now, please see our [talk about this at ReactConf 2019](https://www.youtube.com/watch?v=Tl0S7QkxFE4&list=PLPxbbTqCLbGHPxZpw4xj_Wwg8-fdNxJRh&index=15&t=0s) and [accompanying blog post](https://reactjs.org/blog/2019/11/06/building-great-user-experiences-with-concurrent-mode-and-suspense.html).

**Comment by outerlook** [comment]
Thanks, @josephsavona

**Comment by dohaicuong** [comment]
> Hello any updates on docs to this or any code snippets ?

I managed to put together a working example with entrypoint if you need some reference.
https://github.com/dohaicuong/relay-times/tree/master/relay-entrypoint/src/features/posts

**Comment by steinybot** [comment]
Duplicate here: https://github.com/facebook/relay/issues/3214
